main/src/main/java/org/mobilitydata/gtfsvalidator/cli/Main.java

```diff
@@ -76,7 +78,8 @@ public static void main(String[] argv) {
                 gtfsInput = GtfsInput.createFromPath(Paths.get(args.getInput()));
             }
         } catch (IOException | URISyntaxException | InterruptedException e) {
-            e.printStackTrace();
+            // FIXME: Add system errors to noticeContainer here.
```

Comment:
Could you open an issue pointing to this `FIXME` with more context if it's not going to be implemented in this PR?

Reply:
It's open here: https://github.com/MobilityData/gtfs-validator/pull/602

Reply:
Thanks!